### PR TITLE
update the branding for 5.0.2

### PR DESCRIPTION
**Pull request by Anipik**
- branding updates, pcks already contains the 5.0.1 in packageindex

@ericstj whats the correct entry in the package index

"5.0.0.0": "5.0.0" or "5.0.0.0":"5.0.0.2" ? what are the rules here

### CODE REVIEW

**Comment by ericstj** [comment]
> @ericstj whats the correct entry in the package index

> "5.0.0.0": "5.0.0" or "5.0.0.0":"5.0.0.2" ? what are the rules here


These indicate which assembly version appears in which package version.  Left side is assembly version, right side is package version `AssemblyVersionInPackageVersion`

File: src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>5.0.1</PackageVersion>
-    <AssemblyVersion>5.0.0.1</AssemblyVersion>
```
Comment (ericstj) [inline src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props:5]: Why is this showing as a diff here?  We can't go backwards in AssemblyVersion.  Was this shipped this way?  I thought you had reverted the AssemblyVersions in the shipping package.

Comment (Anipik) [reply src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props:5]: We were late in the cycle so we decided to fix this in 5.0.2

**Comment by Anipik** [comment]
> These indicate which assembly version appears in which package version. Left side is assembly version, right side is package version AssemblyVersionInPackageVersion


we can have only one entry corresponding to 5.0.0.0 so should we update this to 5.0.2 or keep it to 5.0.0

**Comment by Anipik** [comment]
cc @akoeplinger can u take a look at the mono failure, it seems legit. @maryamariyan it seems related to the dependencyInjection change in 5.0.1

File: src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props (hunk 1 above)
Comment (ViktorHofer) [reply src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props:5]: @Anipik just to double check, do you intend to ship the 5.0.2 version of this package with 5.0.0.0 as the assembly version even though the 5.0.1 version of the package had an assembly version of 5.0.0.1? Sounds concerning to me.

Comment (ericstj) [reply src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props:5]: Confirmed that this shipped with the incremented version, we have to keep it:
```
// C:\Users\erics\AppData\Local\Temp\Temp1_microsoft.extensions.dependencyinjection.5.0.1.nupkg\lib\net5.0\Microsoft.Extensions.DependencyInjection.dll
// Microsoft.Extensions.DependencyInjection, Version=5.0.0.1, Culture=neutral, PublicKeyToken=adb9793829ddae60
```

**Comment by akoeplinger** [comment]
The mono failure is addressed with https://github.com/dotnet/runtime/pull/45887 so you can ignore it.

**Comment by maryamariyan** [comment]
@Anipik noticed failure below from the CI logs:

```c#
Microsoft.Extensions.DependencyInjection.Tests.dll -xml testResults.xml -nologo -nocolor -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing  
  Discovering: Microsoft.Extensions.DependencyInjection.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  Microsoft.Extensions.DependencyInjection.Tests (found 99 of 429 test cases)
  Starting:    Microsoft.Extensions.DependencyInjection.Tests (parallel test collections = on, max threads = 2)
    Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteFactoryTest.CreateCallSite_Throws_IfClosedTypeDoesNotSatisfyAbstractClassGenericConstraint [FAIL]
      System.NullReferenceException : Object reference not set to an instance of an object
      Stack Trace:
        /_/src/mono/netcore/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.cs(384,0): at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
  Finished:    Microsoft.Extensions.DependencyInjection.Tests
=== TEST EXECUTION SUMMARY ===
   Microsoft.Extensions.DependencyInjection.Tests  Total: 214, Errors: 0, Failed: 1, Skipped: 0, Time: 0.434s
```

investigating this now locally

**Comment by maryamariyan** [comment]
> building your branch locally now to check for repro.

@Anipik the test didn't fail locally.

**Comment by safern** [comment]
> I'm not running on mono, so couldn't reproduce the failure locally.

You could run on mono by building with

```
build.cmd mono+libs -rc Release
```

Then run the tests as usual but pass down `/p:RuntimeFlavor=mono`.

**Review by ericstj — APPROVED** [review]

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**